### DISCUSSION TITLE
What is the latest version?

### DISCUSSION THREAD

**Discussion by comfriends** [mindsdb/mindsdb#1262] [Get help]
version.json content.

    "lightwood": "0.50.0",
    "mindsdb": "2.14.0",
    "mindsdb_native": "2.24.0",
    "python": "3.8.3 (default, May 19 2020, 18:47:26) [GCC 7.3.0]"

**Comment by ZoranPandovski** [comment]
Hi @combff, by looking at the version JSON you have outdated versions. You should upgrade to the latest mindsdb which is 2.31.0

**Reply by comfriends** [reply]
Is it the latest version on the Docker Hub?
I downloaded it yesterday.
Please check

**Reply by ZoranPandovski** [reply]
I just deployed a new version. Please, run `docker pull mindsdb/mindsdb` again.

**Reply by comfriends** [reply]
![캡처](https://user-images.githubusercontent.com/62046692/119283792-0c65e080-bc79-11eb-86b6-026a8456fdd1.PNG)

![캡처1](https://user-images.githubusercontent.com/62046692/119283810-15ef4880-bc79-11eb-936c-21079086d773.PNG)

There is no match. Is this correct?

**Reply by ZoranPandovski** [reply]
What is the version at the end after `Uninstalling MindsDB-2.14.0` ?
![Screenshot from 2021-05-24 15-41-19](https://user-images.githubusercontent.com/7192539/119356503-c9bafd00-bca6-11eb-89ae-70bd208889ce.png)

**Reply by ZoranPandovski** [reply]
Hello @comfriends did you resolve the issues?
